src/main/java/uk/gov/companieshouse/efs/api/events/service/SameDayServiceDelayedHandler.java

```diff
@@ -29,14 +29,18 @@
     private SubmissionRepository repository;
     private EmailService emailService;
     private int supportDelayInMinutes;
+    private String businessEmail;
 
     @Autowired
     public SameDayServiceDelayedHandler(final SubmissionRepository repository,
         final EmailService emailService,
-        @Value("${submission.sameday.support.minutes}") final int supportDelayInMinutes) {
+        @Value("${submission.sameday.support.minutes}") final int supportDelayInMinutes,
+        @Value("${internal.sharecapitalreduction.sh19.sameday.email.address}")
```

Comment:
do you think it ought to say 'business' in the property name to make it clear which one this is for?

Reply:
ah scrap that - it says 'internal', I missed that at the start.

Reply:
To justify the names:
- `submission.sameday.support.minutes` is the delay when support is notified (and business gets a copy)
- `internal.sharecapitalreduction.sh19.sameday.email.address` is named similar to `internal.sharecapitalreduction.email.address`

Reply:
yeah ok, suppose it just shows that the setup for the different emails isn't trivial.